Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
 name = "coredump_bot"
-version = "0.3.0"
+version = "0.3.1"
 authors = [
   "Stefan Schindler <stefan@estada.ch>"
 ]
 
 [dependencies]
 log = "^0.3"
 env_logger = "^0.3"
-telegram-bot = "^0.4"
+telegram-bot = "^0.5"
 #telegram-bot = { git = "https://github.com/dns2utf8/telegram-bot.git", branch = "api_2.0" }
-hyper = "^0.8"
+hyper = "^0.9"
```

Comment:
Shouldn't we set the version to 0.4.0 if we update dependencies to include breaking changes?